README.md

```diff
@@ -77,7 +77,7 @@ git clone https://github.com/your-username/fedimint.git
 Then enter the nix developer environment.
 
 ```bash
-nix develop
+nix --extra-experimental-features nix-command --extra-experimental-features flakes develop
```

Comment:
We should change the installer recommendation to one from https://github.com/DeterminateSystems/nix-installer , so the users don't need to worry about enabling flakes. It is generally considered much better than the official one, and

Reply:
Hmm, that's even better I suppose. So should I make the appropriate changes in the Nix Installation section?

Reply:
Go for it. Thanks!